docs/processing_information.md

```diff
@@ -87,12 +87,14 @@ When cells were [filtered based on RNA-seq content](#filtering-cells) after quan
 ### Processed ADT data
 
 An ambient profile representing antibody-derived tag (ADT) proportions present in the ambient solution is calculated from the `_unfiltered.rds` object using [`DropletUtils::ambientProfileEmpty()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/ambientProfileEmpty.html).
```

Comment:
Using the file name seemed odd here, I just noticed. We talk about the `.rds` files as output, which makes sense, but I think when we are referring to calculations we should avoid this.


```suggestion
An ambient profile representing antibody-derived tag (ADT) proportions present in the ambient solution is calculated from the unfiltered `SingleCellExperiment` object using [`DropletUtils::ambientProfileEmpty()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/ambientProfileEmpty.html).
```

Reply:
Noting another spot like this was fixed here - 259a000